recipes/inform/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "inform" %}
+{% set version = "1.26.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inform-{{ version }}.tar.gz
+  sha256: 7d5800b556f454a1b75c604d621abce0a3e1cdd43a9038e8e14f39a8211d7e82
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=2.0
+    - python !=3.0,!=3.1,!=3.2
```

```suggestion
    - python >=3.5
```
Otherwise you need to include pathlib for 2.7 which is not possible anymore as conda-forge doesn't build for these versions.